me.kozec.syncthingtk.yml

```diff
@@ -0,0 +1,79 @@
+app-id:              me.kozec.syncthingtk
```

Comment:
Where did you get that app-id from? It doesn't seem to resolve via DNS; is it used internally by the application?

Reply:
:+1: